prow/sidecar/run.go

```diff
@@ -190,9 +190,6 @@ func (o Options) doUpload(spec *downwardapi.JobSpec, passed, aborted bool, metad
 		// TODO(fejta): JobVersion,
 	}
 
-	// TODO(fejta): move to initupload and Started.Repos, RepoVersion
-	finished.Revision = downwardapi.GetRevisionFromSpec(spec)
```

Comment:
Use `finished.DeprecatedRevision` like you did for started.json.

Reply:
Added, thanks Cole